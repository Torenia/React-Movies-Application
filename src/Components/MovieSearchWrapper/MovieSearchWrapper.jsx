import styled from 'styled-components';

export const MovieSearchWrapper = styled.div`
  height: 300px;
  text-align: right;
  background: rgb(66,66,66);
  background: linear-gradient(90deg, rgba(66,66,66,1) 1%, rgba(37,37,90,1) 54%, rgba(66,66,66,1) 100%);
  
  button {
    color: #f65261;
    background-color: #555;
    font-size: 15px;
    text-transform: uppercase;
    border: none;
    border-radius: 5%;
    outline:none;
    cursor: pointer;
    padding: 10px 20px;
    margin: 20px 55px 0 0;
  }
    
  h1 {
    color: #fff;
    text-transform: uppercase;
    text-align: left;
    font-weight: normal;
    margin: 40px 0 50px 110px;
  }
`;
