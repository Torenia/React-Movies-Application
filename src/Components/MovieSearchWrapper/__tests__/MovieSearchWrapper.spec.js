import React from 'react';

import { MovieSearchWrapper } from '../MovieSearchWrapper';

describe('<MovieSearchWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MovieSearchWrapper />);
  });

  it('should render a movie search wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
