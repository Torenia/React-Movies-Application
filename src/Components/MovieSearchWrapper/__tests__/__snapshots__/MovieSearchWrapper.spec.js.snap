// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieSearchWrapper /> should render a movie search wrapper 1`] = `
.c0 {
  height: 300px;
  text-align: right;
  background: rgb(66,66,66);
  background: linear-gradient(90deg,rgba(66,66,66,1) 1%,rgba(37,37,90,1) 54%,rgba(66,66,66,1) 100%);
}

.c0 button {
  color: #f65261;
  background-color: #555;
  font-size: 15px;
  text-transform: uppercase;
  border: none;
  border-radius: 5%;
  outline: none;
  cursor: pointer;
  padding: 10px 20px;
  margin: 20px 55px 0 0;
}

.c0 h1 {
  color: #fff;
  text-transform: uppercase;
  text-align: left;
  font-weight: normal;
  margin: 40px 0 50px 110px;
}

<div
  className="c0"
/>
`;
